### PR TITLE
Update introduction page content

### PR DESCRIPTION
**Pull request by cehsu**
These changes update the chapter 36 introduction page ([design](https://adhoc.invisionapp.com/share/G9F1A8U7K#/screens)).
One departure from the design is the sign in message. The message used here is the default message used for forms with prefill enabled. If the message in the design should be used instead, that can be updated.

localhost_3001_employment_vocational-rehab-and-employment_application_chapter36_introduction

### CODE REVIEW

**Review by rianfowler — APPROVED** [review]

File: src/js/vre/chapter36/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -5,72 +5,119 @@ import { connect } from 'react-redux';
...
+} from '../../../common/schemaform/save-in-progress/SaveInProgressIntro';
 
 class IntroductionPage extends React.Component {
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
   goForward = () => {
     this.props.router.push(this.props.route.pageList[1].path);
-  }
+  };
```
Comment (jbalboni) [inline src/js/vre/chapter36/components/IntroductionPage.jsx]: I think this can be removed.

**Review by jbalboni — APPROVED** [review]
LGTM, needs @peggygannon's approval.

File: src/js/vre/chapter36/components/IntroductionPage.jsx (hunk 1 above)
Comment (cehsu) [reply src/js/vre/chapter36/components/IntroductionPage.jsx]: Thanks, removed.

**Review by peggygannon — COMMENTED** [review]
@cehsu Hi, Claire, two things: 
1. Can we please remove the following: 
**Learn about other VR&E benefits**
[See what other VR&E benefits you may be eligible for as a dependent](/employment/vocational-rehab-and-employment/family-members/).

2. also, I'm wondering if the top note would be more readable as bullet points, as there is currently lot of text in the blue box. 
Here's what I'm thinking: 
[blue box]
If you're signed in to your account, your application process can go more smoothly. Here’s why:
*We can prefill part of your application based on your account details.
*You can save your form in progress, and come back later to finish filling it out. You have 60 days from the date you start or update your application to submit the form. After 60 days, the form won't be saved, and you'll need to start over. 
[end blue box]

**Comment by cehsu** [comment]
Thanks much @peggygannon, I'll update these items. Should the same change be made for the chapter 31 intro page that @rianfowler has been working on ([PR](https://github.com/department-of-veterans-affairs/vets-website/pull/7308))?

**Comment by cehsu** [comment]
Thanks @peggygannon, screenshot updated above.

**Comment by stacyhlinn** [comment]
I don't remember now how we ended up with the DD214 under Prepare... I'm going to mark this as a question for the stakeholder, since we do require certain discharge status' for being eligible, but we never talked about actually having a doc upload chapter for DD214.
I'm going to approve this PR for now and we can update when we find out more.

**Comment by peggygannon** [comment]
@cehsu OK, thank you for the updated screenshot. 

@stacyhlinn I don't believe the VRE intro pages have been shown to stakeholders yet for their feedback.

**Review by jbalboni — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

**Pull request merged**